pydatastructs/linear_data_structures/arrays.py

```diff
@@ -48,8 +48,8 @@ class OneDimensionalArray(Array):
     Examples
     ========
 
-    >>> from pydatastructs import OneDimensionalArray as ODA
```

Comment:
Maybe doctest is failing because u removed the import.

Reply:
This needs to be restored.